assignment.js

```diff
@@ -57,14 +67,28 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
 
+// create array
+// add numbers in array
+// find average of numbers
+// return numbers
+
+let problemNumbers = [1, 2, 3]
+let total = 0
+
+for (i = 0; i < problemNumbers.length; i++) {
+  total += (problemNumbers[i]) / problemNumbers.length
```

Comment:
🛠 Read the comments and update your code accordingly

@nurisellie nice approach, the only issue here is the name of your variable `total`, based on your approach it looks like `total` actually represents the `average`.  Seems small, but other developers reading may get confused as they use variable names to give them a sense of what the code is doing. 

Please update the variable name